src/back/space/src/api/user-list.cpp

```diff
@@ -0,0 +1,56 @@
+#include "user-list.hpp"
+#include "../service/service.hpp"
+#include "../../../shared/headers.hpp"
+#include "../../../shared/errors.hpp"
+#include "../../../shared/paging.hpp"
+#include "../../../shared/paging.сpp"
+#include "../model/user_serialize.hpp"
+
+namespace svetit::space::handlers {
+
+UserList::UserList(
+	const components::ComponentConfig& conf,
+	const components::ComponentContext& ctx)
+	: server::handlers::HttpHandlerJsonBase{conf, ctx}
+	, _s{ctx.FindComponent<Service>()}
+{}
+
+formats::json::Value UserList::HandleRequestJsonThrow(
+	const server::http::HttpRequest& req,
+	const formats::json::Value& body,
+	server::request::RequestContext&) const
+{
+	formats::json::ValueBuilder res;
+
+	const auto& userId = req.GetHeader(headers::kUserId);
+	if (userId.empty()) {
+		res["err"] = "Access denied";
+		req.SetResponseStatus(server::http::HttpStatus::kUnauthorized);
+		return res.ExtractValue();
+	}
+
+	const auto& spaceId = req.GetArg("spaceId");
+
+	try {
```

Comment:
Кажись GetArg может кидать исключение, так что лучше его перенести внутрь trt
```suggestion
	try {
		const auto& spaceId = req.GetArg("spaceId");

```

Reply:
Готово (везде).